CHANGELOG

```diff
@@ -1,3 +1,9 @@
+Release 103 (2019-05-03)
+------------------------
+Add com.snowplowanalytics.mobile/screen/jsonschema/1-0-0 (#866)
+Add com.snowplowanalytics.mobile/screenview/jsonschema/1-0-0 (#919)
```

Comment:
Event is called screen_view not screenview

Reply:
Fixed!